monkey/monkey_island/cc/ui/src/components/report-components/AttackReport.js

```diff
@@ -62,8 +63,10 @@ class AttackReport extends React.Component {
         return 'collapse-warning';
       case ScanStatus.USED:
         return 'collapse-danger';
-      default:
-        return 'collapse-default';
```

Comment:
We should keep default so that the report doesn't crash on other people's forks/tests of Monkey